clientv3/client.go

```diff
@@ -200,5 +200,6 @@ func dialEndpointList(c *Client) (*grpc.ClientConn, error) {
 }
 
 func isRPCError(err error) bool {
-	return grpc.Code(err) != codes.Unknown
+	return grpc.Code(err) != codes.Unknown &&
```

Comment:
the commit message says `treat closing client as RPC error`.

But here we treat grpc.ErrClientConnClosing as !RPCError. I think the code is correct.